src/qforte/ucc/spqe.py

```diff
@@ -106,17 +107,16 @@ def run(self,
             self._pool_obj.add_term(0.0, self.get_op_from_basis_idx(I))
 
         self.build_orb_energies()
-        spqe_iter = 0
-        hit_maxiter = 0
+
+        self._spqe_iter = 1
```

Comment:
Why is this now a member variable?

Reply:
I decided instead of passing it on the few functions that need it, to make it a member variable. If you prefer passing it, I can do that.

Reply:
At present, I don't see any functions that need it. If another PR is going to add them, I can keep it as a member variable for now.